superbench/common/utils/__init__.py

```diff
@@ -9,7 +9,7 @@
 from superbench.common.utils.lazy_import import LazyImport
 from superbench.common.utils.process import run_command
 from superbench.common.utils.topo_aware import gen_topo_aware_config
-from superbench.common.utils.gen_traffic_pattern_config import gen_tarffic_pattern_host_group
+from superbench.common.utils.gen_traffic_pattern_config import (gen_traffic_pattern_host_group, gen_ibstat)
```

Comment:
are the () needed?

Reply:
After merging main branch, more functions will be imported, so it is needed.